Spryker analytics (#2880)
spryker/spryker-docs: docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/best-practices-analyzing-shop-data.md

---

> title: "Best practices: Analyzing shop data"

Reviewer: "Working with Analytics" means editing options. "Analyzing shop data" could also be the use case for viewing the dashboards, which is obviously not part of this section.

description: Learn how to analyze and present shop data.
last_updated: Oct 24, 2024
template: back-office-user-guide-template
related:
---

> Amazon Quicksight is a powerful business intelligence tool. It's highly customizable and has a lot of tools that can't be covered in one document. This document describes a typical use case and how you can start working with analytics, but doesn't cover all the editing options.

Reviewer: ```suggestion
Amazon QuickSight is a powerful business intelligence tool. It's highly customizable and has many tools that can't be covered in one document. This document describes a typical use case and how to start working with analytics, but it doesn't cover all the editing options.
```

> For more insights into creating analyses and dashboards, see [Author Workshop](https://catalog.workshops.aws/quicksight/en-US/author-workshop).

Reviewer: let's remove it please as you give this link in the Conclusion section where it fits better. At the beginning we need to describe our points first and then have this reference for external docs.

## Prerequisites

[Set up Amazon QuickSight](/docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/set-up-amazon-quicksight.html)

> ## Optional: Back up the default analysis

Reviewer: ```suggestion
## Back up the default analysis
```

> The Analytics feature is shipped with a default analysis that can be useful while you're learning how to create your own. If you need to play around with the existing analysis, we recommend duplicating the default one as follows:

Reviewer: ```suggestion
The Analytics feature is shipped with a default analysis that is used for default dashboards. We recommend duplicating the default analysis so you can learn how to create your own dashboards. Please do the following:
```

1. In the Back Office, go to **Analytics**.
2. On the **Analytics** page, click **Analyses**.

> 3. Click on **MASTER Analysis**.

Reviewer: ```suggestion
3. Click on **B2C, B2B, B2C MP or B2B MP Analysis** (based on your business case).
```

4. On the page of the analysis, click **File**>**Save as Analysis**.
5. In the **Save a copy** window, enter an **Analysis name**.
6. Click **SAVE**.

>   This duplicates the analysis and opens it. Now you can safely play around with this duplicate or the original analysis.  

Reviewer: ```suggestion
  This duplicates the analysis and opens it. You can now safely create your own dashboards based on this analysis.  
```

> ## Refresh data

Reviewer: here we need to ultimately mention that it makes sense to set up a scheduler first. No one will go and do it manually. But this process we already described on a separate page. Not sure why we would need this section again…

In this example, you're going to create an analysis with product availability data. Because availability is constantly changing, it's important for data to be up to date.

To refresh product related datasets, do the following:
1. On the **Analytics** page, go to **Datasets**.
2. In the **Datasets** section, click on the **Product Concrete Availability** dataset.
  This opens the dataset's details.
3. Click the **Refresh** tab.
4. Click **REFRESH NOW**.
5. In the **Refresh type** window, select **Full refresh**.
6. In the **Confirm refresh** window, click **REFRESH**
  This refreshes the page. A new refresh entry is displayed in the **History** pane. You can check the status of the refresh in the **Status** column.
7. To go back to the list of datasets, cilck **< Datasets**.

> 8. Repeat steps 2-7 for all the other product datasets, like **Product Concrete + Store**

Reviewer: this one looks scary like the user has to go to 2-7 different datasets and manually update them.

Now the data is fresh and you can use it to create an anlysis.

## Create an analysis

Analysis is basically a visualization of data in your datasets. To create one, follow the steps:

1. On the **Analytics** page, go to **Analyses**.
2. In the **Analyses** section, click **New analysis**.
3. In the **Your Datasets** section, click on the **Order Totals (Custom SQL)** dataset.
  This opens the dataset's details.
4. Click **USE IN ANALYSIS**.
  This creates an analysis called **Order Totals (Custom SQL)** with one visual.

### Edit the visual

In this example, you're going to create a product availability visual:

1. To select the visual, click on it.

> 2. In the **Data** pane, click on **Availability**.

Reviewer: Let's add a new subsection here about calculated fields such as "Availability" in this example. Maybe we can take example for money fields as all of them needs to be adjusted like "{grand_total}/100" to represent the correct number. Example is field in "Total Sales" visual. please check it and describe it as an example. 

More details: https://docs.aws.amazon.com/quicksight/latest/user/adding-a-calculated-field-analysis.html

5. To add filter controls, click the three dots next to the filter and select **Add control**>**Top of this sheet**.
  This adds the filter controls to the top of the sheet.

![store-filter](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/business-intelligence/amazon-quicksight-third-party-integration/best-practices-analyzing-shop-data.md/store-filter.png)  

## Share the analysis

If you want another author to collaborate with you on the analysis, you can share it.

To share an analysis, follow the steps:

1. On the page of the analysis, go to **File** > **Share**.
2. In the **Share analysis**, enter and select the email address of the author to share the analysis with.
  This displays the user in the table.
3. Repeat the previous step until you've added all the authors you want to share the analysis with.
4. Click **Share**
  This closes the window. The added users can now edit the analysis.

{% info_block infoBox "Manage permissions" %}

You can check and manage permissions of all users to an analysis in **File**>**Share**>**Manage analysis permissions**.

{% endinfo_block %}

## Publish analysis as a dashboard and share

> Most of your analytics users will have view permissions. When you're ready to share the analysis with a wider audience, you need to publish it as a dashboard. Users with view permissions can access only dashboards.

Reviewer: ```suggestion
Most of your analytics users will be readers and have view permissions for dashboards. When you're ready to share the analysis with a wider audience, you must publish it as a dashboard so that the readers can access it.
```

To publish and share a dashboard, follow the steps:

1. To publish a dashboard, on the page of an analysis click **PUBLISH**.
2. For **Publish new dashboard as**, enter a name for the dashboard.
3. To confirm, click **Publish dashboard**.
  This publisher the dashboard and opens its page.

![published-dashboard](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/business-intelligence/amazon-quicksight-third-party-integration/best-practices-analyzing-shop-data.md/published-dashboard.png)

4. To share the dashboard, in the top-right corner, click the share icon and select **Share dashboard**.
  This opens the **Share dashboard** page.
5. In the **Invite users and groups to dashboard** pane, start entering the email address of a user.
  This shows users with matching email addresses as you're typing.
6. Next to the needed user, click **ADD** and select the permissions to assign to them.
  This shows the user in the **Manage permissions** pane with the permissions you've assigned.

> ## Conclusion

Reviewer: ```suggestion
## Additional ressources
```

> In this guide, you've followed a typical user journey of creating, collaborating, and presenting data in a meaningful way. Now you can play around with these tools or learn by following [Author Workshop](https://catalog.workshops.aws/quicksight/en-US/author-workshop).

Reviewer: let's please this workshop here as well please:
https://explore.skillbuilder.aws/learn/course/external/view/elearning/14908/getting-started-with-amazon-quicksight
